app/views/automated_tests/_form.html.erb

```diff
@@ -7,7 +7,9 @@
   <script>
     document.addEventListener('DOMContentLoaded', () => {
       window.autotestManagerComponent = makeAutotestManager(document.getElementById('autotest-manager'),
-        { assignment_id: <%= @assignment.id %>});
+        { assignment_id: <%= @assignment.id %>,
+          grouping_id: <%= @test_grouping.id %>
+        });
```

Comment:
This is a really picky comment but... be consistent with where you place the brackets. See the default style (symmetrical) here: https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/MultilineHashBraceLayout